Source/WebInspectorUI/UserInterface/Views/OverrideDeviceSettingsPopover.js

```diff
@@ -0,0 +1,321 @@
+        if (InspectorFrontendHost.isRemote)
+            contentElement.appendChild(this._createUserAgentSection());
+
+        if (WI.DeviceSettingsManager.supportsSetScreenSizeOverride())
+            contentElement.appendChild(this._createScreenSizeSection());
+
+        contentElement.appendChild(this._createSettingsSection());
+        contentElement.appendChild(WI.ReferencePage.DeviceSettings.createLinkElement());
+
+        return contentElement;
+    }
+
+    // Private
+
+    _createSettingsSection()
+    {
+        const coreSettingsGroup = {
+            name: WI.UIString("Core Features"),
+            settings: [
+                {name: WI.UIString("Disable Images"), setting: InspectorBackend.Enum.Page.Setting.ImagesEnabled, value: false},
+                {name: WI.UIString("Disable CSS"), setting: InspectorBackend.Enum.Page.Setting.AuthorAndUserStylesEnabled, value: false},
+                {name: WI.UIString("Disable JavaScript"), setting: InspectorBackend.Enum.Page.Setting.ScriptEnabled, value: false},
+            ],
+        };
+
+        const compatibilitySettingsGroup = {
+            name: WI.UIString("Compatibility"),
+            settings: [
+                {name: WI.UIString("Disable site-specific hacks"), setting: InspectorBackend.Enum.Page.Setting.NeedsSiteSpecificQuirks, value: false},
+            ],
+        };
+
+        const securitySettingsGroup = {
+            name: WI.UIString("Security"),
+            settings: [
+                {name: WI.UIString("Disable cross-origin restrictions"), setting: InspectorBackend.Enum.Page.Setting.WebSecurityEnabled, value: false},
+            ],
+        };
+
+        const privacySettingsGroup = {
+            name: WI.UIString("Privacy"),
+            settings: [
+                {name: WI.UIString("Enable Intelligent Tracking Prevention debug mode"), setting: InspectorBackend.Enum.Page.Setting.ITPDebugModeEnabled, value: true},
+            ],
+        };
+
+        // COMPATIBILITY (iOS 14.0): `Page.Setting.AdClickAttributionDebugModeEnabled` was renamed to `Page.Setting.PrivateClickMeasurementDebugModeEnabled`.
+        if (InspectorBackend.Enum.Page.Setting.PrivateClickMeasurementDebugModeEnabled)
+            privacySettingsGroup.settings.push({name: WI.UIString("Enable Private Click Measurement debug mode"), setting: InspectorBackend.Enum.Page.Setting.PrivateClickMeasurementDebugModeEnabled, value: true});
+
+        // COMPATIBILITY (iOS 14.0): `Page.Setting.AdClickAttributionDebugModeEnabled` was renamed to `Page.Setting.PrivateClickMeasurementDebugModeEnabled`.
+        if (InspectorBackend.Enum.Page.Setting.AdClickAttributionDebugModeEnabled)
+            privacySettingsGroup.settings.push({name: WI.UIString("Ad Click Attribution debug mode"), setting: InspectorBackend.Enum.Page.Setting.AdClickAttributionDebugModeEnabled, value: true});
```

Comment:
Should the first compatibility comment say that "`Page.Setting.PrivateClickMeasurementDebugModeEnabled` was added"? Or maybe these both share the same COMPATIBILITY comment and the legacy path becomes an `if else (...)`?

Reply:
yeah our usual approach is to use an `else`

Reply:
Done. Used `if / else`